nf_core/lint.py

```diff
@@ -122,10 +129,14 @@ def pf(file_path):
             else:
                 self.warned.append((1, "File not found: {}".format(files)))
 
+        # Load and parse files for later
+        if 'environment.yml' in self.files:
+            with open(os.path.join(self.path, 'environment.yml'), 'r') as fh:
+                self.conda_config = yaml.load(fh)
+
 
     def check_docker(self):
-        """minimal tests only"""
-        logging.debug('Checking Dockerfile')
+        """Check Dockerfile"""
```

Comment:
... for what? The doc string should have a little more information, than the function name. ;)

Reply:
True, I was just copying and pasting here - I think @andreas-wilm wrote this originally ;) But yes, I will go through them all and tidy up a bit.

Reply:
Yeah sorry for being sloppy

Reply:
No problem, just have to be picky with Phil, because everything else was fine ;)

Reply:
`git blame` never forgets.. 😆